tests/cron.test.ts

```diff
@@ -1191,7 +1191,6 @@ describe('cron', () => {
 
 	it('should throw when providing both exclusive parameters timeZone and utcOffset', () => {
 		expect(() => {
-			// @ts-expect-error testing runtime exception
```

Comment:
why remove this line? it should be necessary since we're providing both `timeZone` and `utcOffset`, which TypeScript should treat as an error.

Reply:
in the latest version on `main`, this test effectively raises a TypeScript warning. if it does not anymore in your PR, it means you changed something that broke the type checking on these properties.
I don't really see how, but moving the `errorHandler` definition to `BaseCronJobParams` might fix the issue.

Reply:
Apparently the `@ts-expect-error` error was related to the separate constructor I defined, now that I removed it and it works fine.